changelog/src/main/kotlin/com/ritense/valtimo/changelog/service/ChangelogService.kt

```diff
@@ -52,19 +53,31 @@ class ChangelogService(
                 filename = filename,
                 dateExecuted = Instant.now(),
                 orderExecuted = changesetRepository.getNextOrderExecuted() ?: 0,
-                md5sum = md5sum
+                md5sum = md5sum,
+                checksumType = ChangesetCheckSumType.FILE_HASH
             )
         )
     }
 
     fun deleteChangesetsByKey(key: String?) = changesetRepository.deleteAllByKey(key)
 
-    fun isNewValidChangeset(changesetId: String, md5sum: String): Boolean {
+    fun isNewValidChangeset(changesetId: String, md5sum: String, legacyCheckSum: String): Boolean {
```

Comment:
Technically a breaking change, do we want to worry about this or just accept this?